keps/0034-instance-health.md

```diff
@@ -0,0 +1,43 @@
+---
+kep-number: 34
+short-desc: Provide a way to monitor health of KUDO instance post-deployment
+title: KUDO instance health
+authors:
+  - "@alenkacz"
+owners:
+  - "@alenkacz"
+editor: TBD
+creation-date: 2020-09-24
+last-updated: 2020-09-24
+status: provisional
+---
+
+# KUDO instance health
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+
+[Tools for generating]: https://github.com/ekalinin/github-markdown-toc
```

Comment:
Leftover from the tooling that you used?

Reply:
removed 👍